motion/ruby_motion_query/stylers/ui_button_styler.rb

```diff
@@ -52,6 +52,26 @@ def background_image_highlighted=(value)
         @view.setBackgroundImage value, forState: UIControlStateHighlighted
       end
 
+      def background_image_selected=(value)
+        @view.setBackgroundImage value, forState: UIControlStateSelected
```

Comment:
@squidpunch I see the issue, I needed `backgroundImageForState` and now the tests work!